src/admin/WL_Metabox/WL_Metabox_Field_date.php

```diff
@@ -2,14 +2,37 @@
 
 
 class WL_Metabox_Field_date extends WL_Metabox_Field {
+	
+	/*
+	 * Attribute to distinguish between date formats, inferred from the schema property export type
+	 * 
+	 * @since 3.2.0
+	 */
+	private $date_format;
+	
+	/*
+	 * Boolean flag to decide if the calendar should include time or not
+	 * 
+	 * @since 3.2.0
+	 */
+	private $timepicker;
+	
+	public function __construct($args) {
+		
+		// Call parent constructor
+		parent::__construct($args);
```

Comment:
@pieroit according to the WP PHP Coding standards this must look like

    __construct( $args )